Makefile

```diff
@@ -641,6 +641,9 @@ $(STATUS_CLIENT_APPIMAGE): nim_status_client $(APPIMAGE_TOOL) nim-status.desktop
 		-bundle-non-qt-libs \
 		-exclude-libs=libgmodule-2.0.so.0,libgthread-2.0.so.0 \
 		-verbose=1 \
+		-executable=$(APP_DIR)/usr/lib/libpcsclite.so.1 \
+		-executable=$(APP_DIR)/usr/lib/libpcsclite_real.so.1 \
```

Comment:
I think we dont need to include libs here.

Reply:
yeah this was not needed, one of my many monkey patching attempts, thanks for catching this.